content/calico/stars_policy_demo/create_resources.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Create Resources"
+date: 2018-08-07T08:30:11-07:00
+weight: 1
+---
+
+Before creating network polices, let's create the required resources.
+
+Create a namespace called [stars](https://docs.projectcalico.org/v3.2/getting-started/kubernetes/tutorials/stars-policy/manifests/00-namespace.yaml). We will create frontend and backend services in this namespace in later steps
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/00-namespace.yaml
+```
+
+[Create](https://docs.projectcalico.org/v3.2/getting-started/kubernetes/tutorials/stars-policy/manifests/01-management-ui.yaml) a management-ui namespace, with a management-ui service and [replication controller](https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/) within that namespace.
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/01-management-ui.yaml
+```
+Create [frontend](https://docs.projectcalico.org/v3.2/getting-started/kubernetes/tutorials/stars-policy/manifests/03-frontend.yaml) and [backend](https://docs.projectcalico.org/v3.2/getting-started/kubernetes/tutorials/stars-policy/manifests/02-backend.yaml) services within the stars namespace:
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/02-backend.yaml
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/03-frontend.yaml
+```
+Lastly, [create](https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/04-client.yaml) a client namespace, and a client service for a replication controller:
+
+```
+kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/tutorials/stars-policy/manifests/04-client.yaml
+```
+Check their status, and wait for all the pods to reach the Running status:
+
+```
+$ kubectl get pods --all-namespaces
+NAMESPACE       NAME                                                  READY   STATUS    RESTARTS   AGE
```

Comment:
The output needs to be separated from the command so it's not also copy/pasted. I think there are other examples of "Your output should look similar to this:" type of wording.